Model/ProductExport.php

```diff
@@ -62,7 +63,6 @@ public function __construct(
         \Magento\Swatches\Helper\Data $swatchHelper,
         \Magento\Swatches\Helper\Media $swatchMediaHelper,
         \Magento\Framework\View\Element\BlockFactory $blockFactory,
-        \Magento\Catalog\Model\Product\Gallery\ReadHandler $galleryReadHandler,
```

Comment:
This fails on Magento version 2.0, moved to a separate function which checks if class exists before instantiating, otherwise implementing a workaround.